creator-node/src/services/stateMachineManager/stateMonitoring/findReplicaSetUpdates.jobProcessor.ts

```diff
@@ -278,35 +280,32 @@ const _findReplicaSetUpdatesForUser = async (
         )
         unhealthyReplicas.add(replica.endpoint)
       } else if (unhealthyPeersSet.has(replica.endpoint)) {
-        // Else, continue with conducting extra health check if the current observed node is a primary, and
-        // add to `unhealthyReplicas` if observed node is a secondary
-        let addToUnhealthyReplicas = true
-
-        if (replica.endpoint === primary) {
-          addToUnhealthyReplicas = !(await CNodeHealthManager.isPrimaryHealthy(
-            primary
-          ))
-        }
-
-        if (addToUnhealthyReplicas) {
-          logger.error(
-            `_findReplicaSetUpdatesForUser(): Replica ${replica.endpoint} for user ${wallet} was already marked unhealthy and failed an additional health check if it was primary.`
-          )
-          unhealthyReplicas.add(replica.endpoint)
-        }
```

Comment:
this isPrimaryHealthy check is instead performed below at the end as isNodeHealthyOrGracePeriod so it applies to secondaries as well